dbt-cta/dbt_project.yml

```diff
@@ -41,6 +41,7 @@ models:
         - partner
       +materialized: materialized_view
       +auto_refresh: true
+      +full_refresh: true
```

Comment:
referring [to this other PR](https://github.com/community-tech-alliance/airflow-dags/pull/264) - but i dont think this should be hard coded to true

Reply:
agreed! fixed